libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -521,6 +529,63 @@ bool AC_Fence::pre_arm_check(char *failure_msg, const uint8_t failure_msg_len) c
     return true;
 }
 
+/*
+  get our altitude in the FENCE_ALT_TYPE frame
+  return false if not available
+ */
+bool AC_Fence::get_alt_in_frame(float &alt)
```

Comment:
This is the sort of naming you'd expect if passing in a frame.

```suggestion
bool AC_Fence::get_max_alt(float &alt)
```

Reply:
I think it would be "get_alt_max" wouldn't it?

Reply:
Passing in the frame now so the naming is correct